.github/PULL_REQUEST_TEMPLATE/device_integration.md

```diff
@@ -0,0 +1,68 @@
+<!--
+
+Please name your PR like "target: add device name".
+For example, to add Xiaomi Mi Router 4A (Gigabit Edition) name the PR:
+`ramips-mt7621: add Xiaomi Mi Router 4A`
```

Comment:
I would prefer a less shitty device as an example, also the syntax in the past was mostly "add support for" AFAIK. Why did you leave out the "(Gigabit Edition)"?

```suggestion
Please name your PR like "$TARGET: add support for $DEVICE_NAME".
For example, to add Xiaomi Mi Router 4A (Gigabit Edition) name the PR:
`ramips-mt7621: add support for Xiaomi Mi Router 4A (Gigabit Edition)`
```